src/screens/SUE/SueMapViewScreen.tsx

```diff
@@ -0,0 +1,131 @@
+import { StackNavigationProp } from '@react-navigation/stack';
+import React, { useContext, useEffect, useState } from 'react';
+import { StyleSheet } from 'react-native';
+
+import { SettingsContext } from '../../SettingsProvider';
+import { Map } from '../../components';
+import { colors, normalize } from '../../config';
+
+export const SueMapViewScreen = ({
+  navigation,
+  route
+}: {
+  navigation: StackNavigationProp<any>;
+  route: any;
+}) => {
+  const {
+    calloutTextEnabled,
+    currentPosition,
+    geometryTourData,
+    isMaximizeButtonVisible,
+    isMyLocationButtonVisible,
+    locations,
+    mapCenterPosition,
+    onMapPress,
+    onMarkerPress,
+    onMyLocationButtonPress,
+    selectedPosition,
+    showsUserLocation
+  } = route?.params ?? {};
+
+  const { globalSettings } = useContext(SettingsContext);
+  const { navigation: navigationType } = globalSettings;
+
+  const [position, setPosition] = useState(selectedPosition);
```

Comment:
i was confused that this is named `setPosition` instead of `setSelectedPosition` like in the other file. it does the same, or am i wrong?

Reply:
That's right. Actually, they both have the same function. I only used this nomenclature because we have the `selectedPosition` that comes with the prop. but I'll update the prop's name to `position`.